shard.yml

```diff
@@ -55,7 +55,6 @@ dependencies:
   # For frontends client
   placeos-frontends:
     github: placeos/frontends
-    version: ~> 0.11
```

Comment:
I'd rather the change to the client in `frontends` is versioned. Pins are good here.